esp/services/ws_esdlconfig/ws_esdlconfigservice.cpp

```diff
@@ -1121,6 +1124,9 @@ bool CWsESDLConfigEx::onConfigureESDLBindingLogTransform(IEspContext &context, I
 
 int CWsESDLConfigEx::getBindingXML(const char * bindingId, StringBuffer & bindingXml, StringBuffer & msg)
 {
+    if (m_isDetachedFromDali)
+        throw MakeStringException(-1, "Cannot fetch Binding XML. ESP is currently detached from DALI.");
+
```

Comment:
Is fixing an unused method better than removing the method? My first thought when I saw this method had the only unprotected m_esdlStore dereference was to remove it.

Reply:
If we need to throw an exception here, better use makeStringException().

Reply:
Since this is a private function and unused it may be best to remove as Tim suggested.